### ISSUE TITLE
Warnings in cron-apt log

### ISSUE THREAD

**Issue by garrettr** [freedomofpress/securedrop#852]
```
CRON-APT RUN [/etc/cron-apt/config]: Tue Jan 27 04:00:01 UTC 2015
CRON-APT SLEEP: 784, Tue Jan 27 04:13:05 UTC 2015
CRON-APT ACTION: 0-update
CRON-APT LINE: /usr/bin/apt-get -o quiet=1 update -o quiet=2 -o Dir::Etc::sourcelist=/etc/apt/security.list
W: Duplicate sources.list entry http://104.236.171.240/ trusty/main amd64 Packages (/var/lib/apt/lists/104.236.171.240_dists_trusty_main_binary-amd64_Packages)
W: Duplicate sources.list entry http://deb.torproject.org/torproject.org/ trusty/main amd64 Packages (/var/lib/apt/lists/deb.torproject.org_torproject.org_dists_trusty_main_binary-amd64_Packages)
W: You may want to run apt-get update to correct these problems
```

We should fix these duplicate sources.list entries. I am also still unclear on why we have/need *both* `0-update` and `5-security`.

**Comment by garrettr** [comment]
cron-apt appears to be complaining because the entries in `/etc/apt/security.list` are duplicated in `/etc/apt/sources.list.d`.

**Comment by garrettr** [comment]
Ignore comment about 0-update and 5-security, I get it now.

**Comment by garrettr** [comment]
The entries in `/etc/apt/sources.list.d` are added automatically by Ansible's apt module when it sets up the 3rd party (Tor and FPF) repos during installation. We want to tell cron-apt to only install a subset of updates automatically, so we use `security.list` to specify the repos to auto-update from. This is meant to be a trade-off between always allowing security updates and minimizing potential breakage due to updates. This list includes the custom repos, since we definitely want automatic updates updates from Tor and FPF.

**Comment by dolanjs** [comment]
This was resolved in pr #854  and can be closed.
